### DISCUSSION TITLE
Unnecessary PHP syntax coloring requirement

### DISCUSSION THREAD

**Discussion by mprewittcp** [massCodeIO/massCode#132] [Ideas]
It appears that PHP syntax coloring does not occur unless the code snippet starts with `<?php` and ends with `?>`. This is a bit awkward for code snippets, since a user would not typically be enclosing each block of code with that markup. Is there any way this could be fixed so that the `<?php` and `?>` are optional?

**Comment by antonreshetov** [comment]
In v3 no need to start snippets with `<?php`
![image](https://user-images.githubusercontent.com/17480987/183037642-fb20112a-94ce-4018-8f93-6ef9fab164df.png)
